CMakeLists.txt

```diff
@@ -24,6 +24,11 @@ include( GNUInstallDirs )
 
 get_directory_property( has_parent PARENT_DIRECTORY )
 
+if (POLICY CMP0077)
```

Comment:
This makes it compatible with older CMake versions which do not support this policy.